packages/light-client/test/unit/syncNaive.test.ts

```diff
@@ -1,25 +1,31 @@
 import {expect} from "chai";
-import bls from "@chainsafe/bls";
-import {altair, phase0, Root, ssz, SyncPeriod} from "@lodestar/types";
-import {BeaconStateAltair} from "@lodestar/state-transition";
-import {toHexString} from "@chainsafe/ssz";
-import {chainConfig} from "@lodestar/config/default";
+import bls, {init} from "@chainsafe/bls/switchable";
 import {createIBeaconConfig} from "@lodestar/config";
+import {chainConfig} from "@lodestar/config/default";
 import {EPOCHS_PER_SYNC_COMMITTEE_PERIOD, SLOTS_PER_EPOCH} from "@lodestar/params";
-import {processLightClientUpdate} from "../naive/update.js";
-import {prepareUpdateNaive, IBeaconChainLc} from "../prepareUpdateNaive.js";
-import {getInteropSyncCommittee, getSyncAggregateSigningRoot, SyncCommitteeKeys} from "../utils.js";
+import {altair, ssz, SyncPeriod} from "@lodestar/types";
 import {LightClientStoreFast} from "../../src/types.js";
+import {BeaconChainLcMock} from "../mocks/BeaconChainLcMock.js";
+import {processLightClientUpdate} from "../utils/naive/update.js";
+import {IBeaconChainLc, prepareUpdateNaive} from "../utils/prepareUpdateNaive.js";
+import {getInteropSyncCommittee, getSyncAggregateSigningRoot, SyncCommitteeKeys} from "../utils/utils.js";
+import {isNode} from "../../src/utils/utils.js";
+
+function getSyncCommittee(
+  syncCommitteesKeys: Map<SyncPeriod, SyncCommitteeKeys>,
+  period: SyncPeriod
+): SyncCommitteeKeys {
+  let syncCommitteeKeys = syncCommitteesKeys.get(period);
+  if (!syncCommitteeKeys) {
+    syncCommitteeKeys = getInteropSyncCommittee(period);
+    syncCommitteesKeys.set(period, syncCommitteeKeys);
+  }
+  return syncCommitteeKeys;
+}
 
-/* eslint-disable @typescript-eslint/naming-convention */
-
-describe("Lightclient flow", () => {
-  before("BLS sanity check", () => {
-    const sk = bls.SecretKey.fromBytes(Buffer.alloc(32, 1));
-    expect(sk.toPublicKey().toHex()).to.equal(
-      "0xaa1a1c26055a329817a5759d877a2795f9499b97d6056edde0eea39512f24e8bc874b4471f0501127abb1ea0d9f68ac1"
-    );
-  });
+describe("syncNaive", function () {
```

Comment:
I added the naive name here but I think we can use something more descriptive. Like "sync with in-memory API (no HTTP)"

Reply:
Renamed the test.